docker/symfony/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM php:8
 
-RUN apt update && apt upgrade
-RUN apt install -y wget\
+RUN apt-get update && apt-get upgrade
```

Comment:
it's a detail, but it seems to me that the use of `--no-cache` allows to have lighter images